### ISSUE TITLE
Add latest excel files to fix some misc bugs with the eppa and outlook dashboards

### ISSUE THREAD

**Issue by cypressf** [cypressf/mit-climate-data-viz#110]
From Jennifer:

> I went through both the Outlook and EPPA dashboards and found some things that needed to be fixed in the excel files:
> 
> - [x] Missing color assignments: the Outlook version did not include color mapping for several energy/electricity breakdowns; both versions were missing color mapping for air pollutant emissions breakdowns (as well as Sectoral CO2eq Emissions breakdowns.. see next item)
> - [ ] ~Sectoral CO2eq Emissions: I updated the model code for sectoral emissions to use different sector categories. To have these appear correctly on the dashboards, I need to rerun the scenarios to report the correct new categories (in addition to adding color mapping, and adding it to the query list in the outlook excel files)~
> - [x] In EPPA version CO2 Captured and Stored is broken-- it was missing from the query list in the excel, so hopefully adding it there will solve the problem
> - [x] In Outlook version Electricity Generation--Percent Wind&Solar is broken-- it was missing from the query list in the excel, so hopefully adding it there will solve the problem
>  
> Hopefully this new set of excel files (4 for the EPPA dashboard and 3 for the Outlook dashboard) will address all of these issues, except the sectoral emissions which can't be fixed until I rerun the scenarios.

**Comment by cypressf** [comment]
- [x] `CO2 Captured and stored` appears in the list now, but the sector colors for it are broken
- [x] `NOX` has no color in `Air Pollutant Emissions`

**Comment by cypressf** [comment]
Waiting on model rerun for `Sectoral CO2 Emissions` fix, and for sector colors for `CO2 captured and stored` and `NOX`

**Comment by holak** [comment]
I updated the colormap tab in the EPPA and Outlook excel files to use "NOX" instead of "NOx"-- the lower case x could have been causing there to be no color found?

For the EPPA files I also changed the breakdowns for CO2 Captured and Stored to be "Coal with CCS" and "Gas with CCS" (which already have colors assigned in colormap) (the breakdowns used to  of "Coal CCS" and "Gas CCS"). That will hopefully fix the colors for that.

The updated excel files are in the dropbox "EPPA" and "Outlook" folders: https://www.dropbox.com/sh/47ifwxrka67a8wl/AAD0-o18lrN2IsH4gl1ljtK5a?dl=0
